### ISSUE TITLE
Change to plot_ages() causes error

### ISSUE THREAD

**Issue by cgrandin** [pbs-assess/gfplot#73]
Calling `tidy_ages_raw()` followed by `plot_ages()` causes error:
Error: Aesthetics must be either length 1 or the same as the data (595): fill

**To Reproduce**
```
commercial_samples <- gfdata::get_commercial_samples("arrowtooth flounder")
ac <- tidy_ages_raw(commercial_samples, sample_type = "commercial")
plot_ages(ac)
```

This is caused by commit b708e66db55e9404b1e394ce0a2ce69057edc597
This line: https://github.com/pbs-assess/gfplot/blob/d1c81148e28e1d87be2e96f9fa3d341cfe5a1d26/R/plot-ages.R#L170
the addition of the **fill** clause here causes this. I left it because I wasn't sure what you were doing or if you were in the middle of something. Removing it fixes the problem.

**Comment by cgrandin** [comment]
The same issues applies to `plot_lengths()`

**Comment by seananderson** [comment]
I just pushed a better fix for what I was dealing with. This should resolve the issue. The behaviour of ggplot's manual scales had changed and I had attempted a quick fix. I figured out the issue: all manual scales seem to need named vectors now but only the line colour had names and not the fill. It was looking like this in the synopsis report:
![image](https://user-images.githubusercontent.com/19349/124672026-37904200-de6b-11eb-9edb-77faa15e8888.png)

All fixed now.
